testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -150,20 +231,22 @@ class DensityParameters(object):
     def universe(self):
         return mda.Universe(self.topology, self.trajectory, tpr_resid_from_one=False)
 
+
 class TestDensityAnalysis(DensityParameters):
     def check_DensityAnalysis(self, ag, ref_meandensity,
-                                    tmpdir, runargs=None, **kwargs):
+                              tmpdir, runargs=None, **kwargs):
         runargs = runargs if runargs else {}
         with tmpdir.as_cwd():
-            D = density.DensityAnalysis(ag, delta=self.delta, **kwargs).run(**runargs)
+            D = density.DensityAnalysis(
+                ag, delta=self.delta, **kwargs).run(**runargs)
```

Comment:
IMHO this is visually worse than what it was before. If it's to avoid PEP8 issues, I'd at least space indent the arguments here.